nwb_web_gui/dashapps/pages/converter/converter_utils/forms.py

```diff
@@ -157,15 +157,14 @@ def __init__(self, fields, key_name, form_style, definitions=None, composite_chi
 
         self.style = {'margin-top': '1%'}
 
-    def iter_composite_form(self, fields, forms=[]):
+    def iter_composite_form(self, fields, forms=None):
 
+        if forms is None:
+            forms = []
```

Comment:
You should try to avoid using mutables for default values. It can have unintended behavior. This is a better way to accomplish the same thing.